.solhint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"plugins": ["prettier"],
+	"rules": {
+	  "prettier/prettier": "warning"
```

Comment:
I would like to know whether we prefer to display errors when this check fails over warnings.